infra/identity/prod/locals.tf

```
@@ -0,0 +1,19 @@
+locals {
+  prefix    = "io"
+  env_short = "p"
+  env       = "prod"
+  location  = "italynorth"
+  project   = "${local.prefix}-${local.env_short}"
+  domain    = "dev-portal-backend"
```

I suggest
```suggestion
  domain    = "devportal-backend"
```

To use `devportal` as `domain`, and `backend` a `app_name` (I'm referring to Azure resources)